### DISCUSSION TITLE
Pathway contribution to total body clearance

### DISCUSSION THREAD

**Discussion by harridickinson** [Open-Systems-Pharmacology/Forum#786] [Questions & Problems] [answered]
We are trying to build a model that describes the elimination pathways of a compound which has low clearance in vivo.  There is evidence of biliary elimination as well as renal excretion of the parent compound.  The compound is a substrate for efflux transporters as determined by MDCK-MDR1 and Caco2 studies, and has very low hepatic metabolic clearance in vitro.  We have been able to model the fraction eliminated in bile and urine (and understand their contribution to overall clearance from the simulation by looking at percent excreted), however, please could you advise if there is a way to model excretion of the compound from blood into the gut via efflux from the enterocytes (and particularly extract the fraction cleared by this pathway from the model / simulation)?  Additionally is there a way to extract the fraction metabolised from the simulation.  Is their a way to create a table of all clearance pathways and there relative contribution to total body clearance?

**Comment by StephanSchaller** [comment]
Dear @harridickinson ,

that is already great progress.
Have you tried implementing an efflux transporter using the protein expression database (e.g. MDR1/Pgp)?
This will put efflux transporters in enterocytes, bile and kidney amongst others.

Regarding displaying the fractional contribution of these transporters to total elimination: 
this is not possible in PK-Sim at this time as this varies with the types and number of transporters implemented and is quite challenging to implement as a generic feature, but there are workarounds e.g. by using the R-package and summing up different pathways and putting them in relation to total flux.

Hope this helps,
Stephan

**Comment by harridickinson** [comment]
Thanks @StephanSchaller for the really quick response,
We have been using the transporters which is really good, how do we extract the amount of compound that has gone into the gut via enterocytes. Also is there an easy way to calculate hepatic metabolic clearance or extract from the simulation the amount of drug removed by hepatic metabolism.  
Many thanks,

Harri

**Comment by prvmalik** [comment]
> how do we extract the amount of compound that has gone into the gut via enterocytes

You could try using the Lumen > Fraction of Oral Drug Mass Absorbed observer.

> is there an easy way to calculate hepatic metabolic clearance or extract from the simulation the amount of drug removed by hepatic metabolism

You can use the Liver > Intracellular > 'DRUG-METABOLITE-COMPLEX' > Fraction of Dose observer if you have a metabolic enzyme set up.

**Comment by harridickinson** [comment]
Thanks @prvmalik, when plotting the fraction excreted in bile/gallbladder is it then available for reabsorption in the lumen if it has the correct properties?

Cheers,
Harri

**Comment by prvmalik** [comment, marked as answer]
Hi Harri,

No, enterohepatic recirculation is not enabled by default. You must enable gall-bladder emptying.

1. Create an 'Event' to enable continuous gall bladder emptying into the duodenum and then subsequent reabsorption 
![Harri](https://user-images.githubusercontent.com/29066045/96276154-9e4d9280-0fa0-11eb-8574-5023b3c80cb6.JPG)

2. Set the EHC continuous fraction to 0.4 or 1.0, whichever you feel is more appropriate for your modeling purposes. I have seen both done. This parameter is available under the 'ADVANCED' menu.
![Harri2](https://user-images.githubusercontent.com/29066045/96276407-f08eb380-0fa0-11eb-9116-88a62df42b61.JPG)

A word of warning: the fraction of oral drug mass absorbed accounts for all the drug that is absorbed: on first administration plus all the drug that is absorbed during EHC. For this reason it can sum over 100%.

**Comment by harridickinson** [comment]
Thanks @prvmalik, just implemented it, was very helpful.

**Comment by AyaSaleh90** [comment]
> is there an easy way to calculate hepatic metabolic clearance or extract from the simulation the amount of drug removed by hepatic metabolism
Hello @prvmalik, is this step is done on the simulation in PK-Sim or in MoBI?? and if possible can you screenshot from where can I get it, as I looked around but I could not get it as well.
Thanks in advance
